homeassistant/components/recorder/core.py

```diff
@@ -719,6 +719,15 @@ def _run(self) -> None:
         if schema_status is None:
             # Give up if we could not validate the schema
             return
+        if schema_status.current_version > SCHEMA_VERSION:
+            _LOGGER.error(
+                "The database schema version %s is newer than the installed version of "
+                "Home Assistant Core supports (%s), either upgrade Home Assistant Core "
```

Comment:
Do we need to mention Core ? 
```suggestion
                "Home Assistant supports (%s), either upgrade Home Assistant "
```

Reply:
I think so, the database is tied to the core version, and we take care to talk about the Home Assistant Core version in other cases.